### DISCUSSION TITLE
Current version of Xcode is missing

### DISCUSSION THREAD

**Discussion by dmelgar** [community/community#28272] [Actions]
I'm trying to start using Github actions for iOS builds.

At this time, the currently available version of Xcode is 13.4.1, yet the latest on GitHub actions appears to be 13.2.1. This latest version is what I see if I `ls /Applications`.
This is not current enough.

Does GitHub actions really not support the current version of Xcode? 
Is there some other process I'm supposed to do to install or access the current version of Xcode?

**Comment by sebastianvarela** [comment]
You can check the available Xcode versions in each macOS runners:
https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#xcode
https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#xcode
https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md#xcode

And late you can use any of the Xcode-select actions to set the version or refer it inside a fastlane's script.
